### DISCUSSION TITLE
Opening my repo in VSCode in browser not working as expected

### DISCUSSION THREAD

**Discussion by jamiet-msm** [community/community#23926] [Enterprise]
<p>There was a lot of excited clamour yesterday on Twitter (mainly prompted by <a href="https://twitter.com/natfriedman/status/1425504734530650114" rel="noopener nofollow ugc">a Nat Friedman tweet</a>) about the ability to browse to a repo in the GitHub web UI, press ‘.’, and the repo would open up in an instance of VSCode running in the browser. Unfortunately when I tried it, it didn’t work. This is what got displayed:</p>
<p><img src="https://user-images.githubusercontent.com/79862277/181080053-1edb89af-79b9-419f-82a0-2e62dcd79b1f.png" alt="image" data-base62-sha1="frb4WAcjfAQMH8I39CPaBo39Sdr" width="431" height="455"></p>
<p>and the error I received in VSCode’s notifications pane was:</p>
<blockquote>
<p>Unable to open ‘dataplatform’: Unable to resolve resource vscode-vfs://github/myorg/dataplatform</p>
</blockquote>
<p>Note that this is not an open-source repo, it is not publicly available (we keep it private by virtue of being GitHub Enterprise customers).</p>
<p>Does anyone know why this might be the case? I’m assuming its because its not an open-source repo but wanted to check.</p>

**Comment by tajmone** [comment]
<p>Didn’t know about this new feature, so I tried it out on one of my repositories and it works like a charm.</p>
<aside class="quote no-group" data-username="jamiet-msm" data-post="1" data-topic="195173">
<div class="title">
<div class="quote-controls"></div>
 jamiet-msm:</div>
<blockquote>
<p>Note that this is not an open-source repo, we are GitHub Enterprise customers.</p>
</blockquote>
</aside>
<p>I don’t think that the license or the GitHub account type matter, but what might affect this is whether the repository is public or not — you didn’t mention it, so is the repository public or private?</p>
<p>As for the account type, I don’t know much about Enterprise and whether it would support the same features or not. But I’d start by focusing on the repository visibility status first.</p>

**Comment by jamiet-msm** [comment]
<p>thx. Its a private repo and I have updated my question to be explicit about that.</p>

**Comment by brasic** [comment]
<p>Hey <a class="mention" href="/jamiet-msm">@jamiet-msm</a>, do you see any obvious errors in your browser console or network tab?</p>
<p>This feature should work for all public and private repositories.  If your user can see the repo on <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> you should also be able to see it on github.dev.</p>

**Comment by kdelee** [comment]
<p>Privacy Badger was causing this same symptom for me. Disabling Privacy Badger and reloading fixed it.</p>
<p>Though I would say if the website could detect it was being blocked and indicate to the user, it would be a better UX than this pretty opaque error</p>

**Comment by jamiet-msm** [comment]
<p>Thanks you <a class="mention" href="/brasic">@brasic</a> . I had a twitter convo with one of your GitHub colleagues and we got to the root cause: <a href="https://twitter.com/srivatsn/status/1425862550776807431?s=20" rel="noopener nofollow ugc">https://twitter.com/srivatsn/status/1425862550776807431?s=20</a>. I’m getting a 403 on <a href="http://vscode-webview.net" rel="noopener nofollow ugc">vscode-webview.net</a>.</p>
<p>Still don’t know what I can do to solve it mind you.</p>

**Comment by sudo-nautilus** [comment]
<aside class="quote no-group" data-username="tajmone" data-post="2" data-topic="195173">
<div class="title">
<div class="quote-controls"></div>
 tajmone:</div>
<blockquote>
<p>Didn’t know about this new feature, so I tried it out on one of my repositories and it works like a charm.</p>
</blockquote>
</aside>
<p>I thought it was an April fool thing. But now I understood.</p>

**Comment by jamiet-msm** [comment, marked as answer]
<p><a class="mention" href="/brasic">@brasic</a> We figured it out. We use zscalar (<a href="https://www.zscaler.com/" rel="noopener nofollow ugc">https://www.zscaler.com/</a>) and it was zscalar that was blocking <a href="https://t.co/VJrEDkVJ3f?amp=1" rel="noopener nofollow ugc">http://vscode-webview.net</a> &amp; <a href="https://t.co/SduFAUeZhl?amp=1" rel="noopener nofollow ugc">http://github.vscode-unpkg.net</a>.</p>
<p>In the words of our zscalar admin:</p>
<blockquote>
<p>if you go to <a href="https://sitereview.zscaler.com/" rel="noopener nofollow ugc">https://sitereview.zscaler.com/</a> you can enter an address to see it’s category, these two had non. I have submitted these for review as we have more aggressive web filtering setup to unknown sites… anyone can submit a review to Zscaler they check these over and once a category is set most things tend to start working correctly as almost all policies we have in Zscaler are built on category.</p>
<p>This is so new domains used in malware/spam/phishing etc are treated with more caution blocking downloads/uploads etc.</p>
</blockquote>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="jamiet-msm" data-post="8" data-topic="195173">
<div class="title">
<div class="quote-controls"></div>
 jamiet-msm:</div>
<blockquote>
<p>We figured it out.</p>
</blockquote>
</aside>
<p>Excellent! You should mark your solution as the solution.</p>
